Docker-Swarm-deployment/single-node/rs-delete-script/delete-script.yaml

```diff
@@ -0,0 +1,36 @@
+version: '3.7'
+
+networks:
+  overlay-net:
+          external: true
+          driver: overlay
+
+services:
+
+  delete-script:
+    image: ghcr.io/datakaveri/rs-delete-subs-script:5.0.0
```

Comment:
Can we have the tag to be 1.0.0 instead of 5.0.0, because its being released first this release.